networkx/algorithms/approximation/tests/test_clique.py

```diff
@@ -1,6 +1,21 @@
-from nose.tools import *
+# test_clique.py - unit tests for the approximation.clique module
+#
+# Copyright 2015 NetworkX developers.
+#
+# This file is part of NetworkX.
+#
+# NetworkX is distributed under a BSD license; see LICENSE.txt for more
+# information.
+"""Unit tests for the :mod:`networkx.algorithms.approximation.clique`
```

Comment:
It doesn't show up rendered anywhere, but this is a habit I have because 1.
it's a good habit to document every module, class, and function, 2. it's
easier to remove documentation than to add it later, if necessary, and 3. I
don't know what the future holds, it may displayed at some point for some
reason.